main.py

```
@@ -0,0 +1,179 @@
+from fastapi import FastAPI, Request, HTTPException
```

Ruff (0.12.2)

`fastapi.Request` imported but unused

Remove unused import: `fastapi.Request`

(F401)